.github/CONTRIBUTING.md

```diff
@@ -4,6 +4,26 @@ Thank you for your interest in WorkflowSchema!
 [![Issues](https://img.shields.io/github/issues/wwt/WorkflowSchema?color=bright-green)](https://github.com/wwt/WorkflowSchema/issues)
 [![Pull Requests](https://img.shields.io/github/issues-pr/wwt/WorkflowSchema?color=bright-green)](https://github.com/wwt/WorkflowSchema/pulls)
 
+## Updating the schema
+If your contribution meets any of the below guidelines it's required that you add the appropriate label on the pull request.
+Using these labels will run a GitHub action that revs the version accordingly.
+
+#### Important note
+Updating the version of the schema requires that all the libraries using the schema are updated to reflect the new version.
+This includes [SwiftCurrent](https://github.com/wwt/SwiftCurrent).
+
+### No label needed
+- metadata changes. `ex. examples are updated, descriptions are revamped, etc.`
+
+### Patch label
+- additive changes `ex. new fields`
+
+### Minor label
+- break to the existing api. `ex. changing the way that FlowRepresentableName works`
```

Comment:
DO MINOR: When a field is made parsable both as a string and as an object
DO MINOR: When a field name is changed
DO MINOR: When a field type is changed

DO NOT MINOR: When we change a fundamental behavior (flowPersistence)
DO NOT MINOR: When `sequence` is no longer what we use to describe workflows